zenml/core/pipelines/training_pipeline.py

```diff
@@ -223,17 +254,20 @@ def get_tfx_component_list(self, config: Dict[Text, Any]) -> List:
     def add_split(self, split_step: BaseSplit):
         self.steps_dict[keys.TrainingSteps.SPLIT] = split_step
 
+    def add_sequencer(self, sequencer_step: BaseSequencerStep):
+        self.steps_dict[keys.TrainingSteps.SEQUENCER] = sequencer_step
+
     def add_preprocesser(self, preprocessor_step: BasePreprocesserStep):
-        self.steps_dict[keys.TrainingSteps.PREPROCESSING] = preprocessor_step
+        self.steps_dict[keys.TrainingSteps.PREPROCESSER] = preprocessor_step
 
     def add_trainer(self, trainer_step: BaseTrainerStep):
-        self.steps_dict[keys.TrainingSteps.TRAINING] = trainer_step
+        self.steps_dict[keys.TrainingSteps.TRAINER] = trainer_step
 
     def add_evaluator(self, evaluator_step: TFMAEvaluator):
-        self.steps_dict[keys.TrainingSteps.EVALUATION] = evaluator_step
+        self.steps_dict[keys.TrainingSteps.EVALUATOR] = evaluator_step
```

Comment:
Oh this small change might cascade a series of merge conflicts I fear

Reply:
That is a change that I applied to some other steps as well. If I recall correctly, in our docs and discussions, we always referred to the steps as `Trainer` instead of `Training` or `Evaluator` instead of `Evaluation`. That is why I tried to make them consistent. I can change it back if it would cause any issues.